kubernetes/helm/README.md

```diff
@@ -0,0 +1,114 @@
+# CKEditor Collaboration Server On-Premises Helm charts
+
+Use this Helm charts to provision CKEditor Collaboration Server on your
+Kubernetes cluster.
+
+**Note**: This script uses Helm. If you haven't worked with it before, we highly
+recommend to get familiar with it first: https://helm.sh/docs/intro/quickstart/
+
+## List of charts
+
+Short description of every helm chart in this directory:
+
+### [ckeditor-cs](ckeditor-cs)
+
+Helm chart containing the resources to run CKEditor Collaboration Server
+configured to connect to external third-party services.
+
+### [ckeditor-cs-development-stack](ckeditor-cs-development-stack)
+
+Fast way to provision whole infrastructure needed for CKEditor Collaboration
+Server. Used charts as a dependencies:
+- `ckeditor-cs`
+- `bitnami/mysql`
+- `bitnami/redis`
+
+>:warning: **Not for production purposes!** This configuration is insecure, not
+>persistent and its purpose is testing only.
+
+## Quick start
+
```

Comment:
IMO it will be good to add here information that this is a Quick Start for production and it required created databases. Or something similar.

Reply:
Or maybe you could move it to `ckeditor-cs/READM.md` and here add links only:

*   Dev Quick Start 
*   Production Quick Start 

Or something similar.